README.md

```diff
@@ -52,9 +50,7 @@ instance of an embedded AMQP broker (https://deno.land/x/lop/mod.ts) and a
 webhook receiver for testing (`testWebhookReceiver.ts`) on http://localhost:8888.
 
 The web component can be started with `pushd web; deno task start; popd` (hot
```

Comment:
since deno.jsonc merged with web/deno.json, just deno task serve-web would be sufficient.
(currently serve-web starts dev.ts, would another entry for main.ts needed?)

Reply:
Amended!